hack/tools/go.mod

```
@@ -52,8 +52,8 @@ require (
 	github.com/xlab/treeprint v0.0.0-20181112141820-a009c3971eca // indirect
 	go.starlark.net v0.0.0-20200306205701-8dd3e2ee1dd5 // indirect
 	golang.org/x/mod v0.14.0 // indirect
-	golang.org/x/net v0.19.0 // indirect
-	golang.org/x/sys v0.15.0 // indirect
+	golang.org/x/net v0.23.0 // indirect
```

```suggestion
	golang.org/x/net v0.24.0 // indirect
```
0.24.0 was out and bumped by dependabot already, so do it here as well to avoid extra PRs.

---

Thanks. It's done now.